wpeverest/user-registration
# Fix - Elementor popup shortcode usage compatibility

### All Submissions:






### Changes proposed in this Pull Request:



Closes #196  .

### How to test the changes in this Pull Request:

Git command:

`git remote add coderkoala https://github.com/coderkoala/user-registration`
```
git fetch coderkoala
git checkout coderkoala/fix/elementor_popup
```

1. Make a popup, implement a UR shortcode
2. Fill in the values and press save
3. Expect the user saving, and submission routines running properly.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Elementor popup shortcode usage compatibility.


##### Before
![image](https://user-images.githubusercontent.com/30156167/72331799-e9203e00-36e0-11ea-939d-2ec6939a2ecf.png)

##### After
![image](https://user-images.githubusercontent.com/30156167/72331673-af4f3780-36e0-11ea-958f-2f316c18edea.png)
![image](https://user-images.githubusercontent.com/30156167/72331767-d574d780-36e0-11ea-9c91-7c80c8ef136f.png)
